packages/playground/tailwind/src/main.js

```diff
@@ -0,0 +1,6 @@
+import { createApp } from 'vue'
+import App from './App.vue'
+import router from './router'
+// import '../index.css';
```

Comment:
Why is this import commented? Shouldn't this be part of the tailwind setup https://tailwindcss.com/docs/guides/vue-3-vite#include-tailwind-in-your-css ?